float64 point_x
float64 point_y
